test/realm_test.dart

```diff
@@ -194,6 +194,112 @@ Future<void> main([List<String>? args]) async {
       config.schemaVersion = 3;
       expect(config.schemaVersion, equals(3));
     });
+
+    test('Configuration isReadOnly property - realm files must exist', () {
+      Configuration config = Configuration([Car.schema]);
+      config.isReadOnly = true;
+      //You cannot open Realm in read-only mode if Realm files do not exist.
```

Comment:
I suggest we remove these comments here. I suggested changing the name of the test which will clarify what the test does.